packages/kbn-apm-synthtrace/src/scenarios/service_many_dependencies.ts

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
I think we can remove this scenario before merging this PR>

Reply:
It's used in the test, right?

Reply:
The test uses an adapted copy of this

Reply:
I see now that we also have `x-pack/plugins/apm/ftr_e2e/cypress/e2e/dependencies/generate_many_dependencies.ts` - makes sense 👍